### DISCUSSION TITLE
Email templates missing variable {{var total}}

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#2121] [Questions & Answers]
I am converting modules/theme from old magento to fresh magento-lts, and noticed that the order_new.html does not have the order total any more.

These all return {{var total}} {{var order.total}} {{var order.grandTotal}} {{var order.GrandTotal}} nothing

In my old magent 1.9.?.? {{var total}} was returning a value. At first I thought I maybe hacked some code of the original magento site, but then I saw this list of available variables that includes this {{var total}}. So I guess it was available by default.

https://www.parenttechnology.com/blog/magento-commerce/237-email-variables-for-magento.html

### Preconditions (*)

magento-lts from git 1.9.4.x

### Steps to reproduce (*)
Just put {{var total}} in order_new.html

### Expected result (*)
value

### Actual result (*)
empty

**Comment by f1-outsourcing** [comment]
Maybe want to have this?

https://github.com/OpenMage/magento-lts/commit/6709c5a3302f5e66529c3026eaa9f7b4020f39f9

**Comment by elidrissidev** [comment]
> Maybe want to have this?
> 
> [6709c5a](https://github.com/OpenMage/magento-lts/commit/6709c5a3302f5e66529c3026eaa9f7b4020f39f9)

I did a blame on those lines and saw that they were never modified during at least the last seven years. I don't really play with the Magento frontend that much but maybe I can help you if you can figure out the version you were in before the upgrade.

**Comment by f1-outsourcing** [comment]
I honestly do not know. It might very well be that this never existed, strange is that I found a website that mentioned this variable. The newer version 2.3 had it at least as order.grandtotal. Here in 1.9 you can also use order.grand_total, but it is not nicely formatted.
